Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataRecordPrivate.h

```diff
@@ -37,7 +37,6 @@ WK_EXTERN NSString * const _WKWebsiteDataTypeCredentials WK_API_AVAILABLE(macos(
 WK_EXTERN NSString * const _WKWebsiteDataTypeAdClickAttributions WK_API_AVAILABLE(macos(10.15), ios(13.0));
 WK_EXTERN NSString * const _WKWebsiteDataTypePrivateClickMeasurements WK_API_AVAILABLE(macos(12.0), ios(15.0));
 WK_EXTERN NSString * const _WKWebsiteDataTypeAlternativeServices WK_API_AVAILABLE(macos(11.0), ios(14.0));
-WK_EXTERN NSString * const _WKWebsiteDataTypeFileSystem WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
```

Comment:
Are there any clients that use this symbol directly? If so, for backwards compatibility / to avoid breaking the build, we might need to keep its definition, and change the string value to the new "WKWebsiteDataTypeFileSystem".

Reply:
Good point! Updated patch to keep the old interface with comments.